README.md

```diff
@@ -48,27 +48,28 @@ plugins {
+| `transferAcceleration`              | Configures [S3 Transfer Acceleration](https://aws.amazon.com/ru/s3/transfer-acceleration/)                                                                    | no        | false                            |
+| `headers`                           | A map with HTTP headers to be added to each request (nulls are ignored). e.g. `[ 'x-header-name': 'header-value' ]`                                           | no        |                                  |
+| `awsAccessKeyId`                    | The AWS access key id                                                                                                                                         | no        | `getenv("S3_BUILD_CACHE_ACCESS_KEY_ID")` |
+| `awsSecretKey`                      | The AWS secret key                                                                                                                                            | no        | `getenv("S3_BUILD_CACHE_SECRET_KEY")` |
+| `sessionToken`                      | The [AWS sessionToken](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_temp_use-resources.html) when you use temporal credentials             | no        | `getenv("S3_BUILD_CACHE_SESSION_TOKEN")` |
+| `awsProfile`                        | The [AWS profile](https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/credentials-profiles.html) to use for authentication                        | no        | `getenv("S3_BUILD_CACHE_PROFILE")` |
+| `lookupDefaultAwsCredentials`       | Configures if `DefaultAWSCredentialsProviderChain` could be used to lookup credentials                                                                        | yes       | false                            | 
+| `showStatistics`                    | Displays statistics on the remote cache performance                                                                                                           | Yes       | `true`                           |
+| `showStatisticsWhenImpactExceeds`   | Specifies minimum duration to trigger printing the stats, milliseconds                                                                                        | Yes       | `100`                            |
+| `showStatisticsWhenSavingsExceeds`  | Specifies minimum duration to trigger printing the stats, milliseconds                                                                                        | Yes       | `100`                            |
+| `showStatisticsWhenWasteExceeds`    | Specifies minimum duration to trigger printing the stats, milliseconds                                                                                        | Yes       | `100`                            |
+| `showStatisticsWhenTransferExceeds` | Specifies minimum transfer size to trigger printing the stats, bytes                                                                                          | Yes       | 10*1024*1024                     |
```

Comment:
Could you add a new row and avoid re-formatting all the old ones?

Reply:
Done, is it better now?